themes/classic/templates/_partials/breadcrumb.tpl

```diff
@@ -33,8 +33,9 @@
               <meta itemprop="position" content="{$smarty.foreach.breadcrumb.iteration}">
             </li>
           {elseif isset($path.title)}
-            <li>
-              <span>{$path.title}</span>
+            <li itemprop="itemListElement" itemscope itemtype="http://schema.org/ListItem">
```

Comment:
```suggestion
            <li itemprop="itemListElement" itemscope itemtype="https://schema.org/ListItem">
```

Reply:
True, this is probably originally from time when schema.org always used http: https://schema.org/docs/faq.html#19

Looks like there are some inconsistencies on the usage around the templates already. 

Maybe another issue and pr for making them all consistent?

Reply:
https://github.com/PrestaShop/PrestaShop/search?q=itemtype%3D%22

7 places using https 
9 using http

Reply:
> Looks like there are some inconsistencies on the usage around the templates already.

Indeed, you are right !

> Maybe another issue and pr for making them all consistent?

Yes you can provide another PR to fix it.

Thanks you @tswfi

Reply:
@PululuK what is the process? should there always be a issue first and then a pr? (this is my second pr to the prestashop project so I'm still trying to understand the process :)

Reply:
Created new issue for this: #22418 pr in a moment.

Reply:
Yeah, every PR needs an issue, thanks for your contribs @tswfi :)